templates/rhpam70-authoring.yaml

```diff
@@ -626,9 +617,9 @@ objects:
           - name: MAVEN_REPO_PATH
             value: "/maven2/"
           - name: MAVEN_REPO_USERNAME
-            value: "${MAVEN_REPO_USERNAME}"
```

Comment:
I understand why it's easier to replace the maven user/pass with the admin user/pass in these lines, because of https://issues.jboss.org/browse/RHDM-319 . However, once that jira is completed, this change will have to get reverted. So, the options are 1) keep it as maven user/pass, and people will have to continue passing in the same args for both admin and maven user/pass until RHDM-319 is fixed, or 2) make these changes here, but then have to revert them to maven user/pass once RHDM-319 is fixed. Thoughts?

Reply:
Understood. Will now merge, and we'll have to circle back this. Thanks!